components/home/Collaboration.tsx

```diff
@@ -28,12 +28,14 @@ const Collaboration: NextPage = () => {
                   key={client.name}
                   className="flex items-center justify-center"
                 >
+                  <a href={client.url} target="_blank">
```

Comment:
same comment as the one in Client file. Also, consider making each company component into a compoenent class? since the old way seems to write it in a repetitive way?